bootstrap/storageclass/helmfile/values/nfs-client-provisioner.yaml.gotmpl

```diff
@@ -1,12 +1,14 @@
-resources: {{- toYaml .Values.nfsProvisioner.resources | nindent 2  }}
+resources:    {{- toYaml .Values.nfsProvisioner.resources | nindent 2  }}
 nodeSelector: {{- toYaml .Values.nfsProvisioner.nodeSelector | nindent 2 }}
-affinity: {{- toYaml .Values.nfsProvisioner.affinity | nindent 2 }}
-tolerations: {{- toYaml .Values.nfsProvisioner.tolerations | nindent 2 }}
+affinity:     {{- toYaml .Values.nfsProvisioner.affinity | nindent 2 }}
+tolerations:  {{- toYaml .Values.nfsProvisioner.tolerations | nindent 2 }}
 
 storageClass:
   defaultClass: true
```

Comment:
This should be set based on config!

Reply:
Yes I agree, i decided to skip it since only nfs has this option(as the other classes are just plain manifests). But sure we can do it for nfs right away.